### ISSUE TITLE
Redux-ORM typescript example

### ISSUE THREAD

**Issue by denis-n** [tomasz-zablocki/redux-orm-typings#1]
Hi,

first, thank you for fantastic job)

I'm using right now the old version of `@types/redux-orm`.

Can you please add a few examples how to use latest version of it?

Dennis

**Comment by tomasz-zablocki** [comment]
Whether it's fantastic remains to be seen 😉 

I'm working on new redux-orm documentation, which - if everyone agrees - will contain detailed redux-orm typescript recipes. 

Let me know if you find [this](https://github.com/tomasz-zablocki/redux-orm-typings/blob/master/types/redux-orm/redux-orm-tests.ts) helpful.

If not, than we'll work something out.

**Comment by denis-n** [comment]
Yes, it is helpful.
Thank you.

**Comment by tiangolo** [comment]
Thanks for this package (in fact, for the version in DefinitelyTyped)!

I was using the tests in this repo as linked above, as a template.

But I was getting some errors. I just realized that the version in DefinitelyTyped seems to be a bit more up to date, and using the examples there, I no longer get the errors (although I found it by hand, and then saw that the tests there are a bit different).

:rocket: :heavy_check_mark: Those tests can probably be a better template: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/redux-orm/redux-orm-tests.ts

I'm adding this comment just in case others end up here (as I did) and need to find a template to use the types.
